test/JDBC/expected/BABEL-2934.out

```diff
@@ -0,0 +1,91 @@
+CREATE TABLE BABEL_2934 (c TIME(7) NULL, d TIME(7) NULL)
+GO
+INSERT BABEL_2934 VALUES ('00:59:59.9999123' , NULL)
+GO
+~~ROW COUNT: 1~~
+
+INSERT BABEL_2934 VALUES ('00:00:01.0000120' , NULL)
+go
+~~ROW COUNT: 1~~
+
+
+-- add 300 nanoseconds (the 7th digit shows as '3')
+update BABEL_2934 set d = dateadd(ns, 300, c)
+go
+~~ROW COUNT: 2~~
+
+select c, d, convert(varchar(30), c, 109) as c2, convert(varchar(30), d, 109) as d2 from BABEL_2934
+go
+~~START~~
+time#!#time#!#varchar#!#varchar
+00:59:59.9999120#!#00:59:59.9999120#!#12:59:59.9999120AM#!#12:59:59.9999120AM
+00:00:01.0000120#!#00:00:01.0000120#!#12:00:01.0000120AM#!#12:00:01.0000120AM
+~~END~~
+
+
+drop table BABEL_2934
+go
+
+CREATE TABLE BABEL_2934 (c DATETIME2(7) NULL, d DATETIME2(7) NULL)
+GO
+INSERT BABEL_2934 VALUES ('00:59:59.9999123' , NULL)
+GO
+~~ROW COUNT: 1~~
+
+INSERT BABEL_2934 VALUES ('00:00:01.0000120' , NULL)
+go
+~~ROW COUNT: 1~~
+
+
+-- add 300 nanoseconds (the 7th digit shows as '3')
+update BABEL_2934 set d = dateadd(ns, 300, c)
+go
+~~ROW COUNT: 2~~
+
+select c, d, convert(varchar(30), c, 109) as c2, convert(varchar(30), d, 109) as d2 from BABEL_2934
+go
+~~START~~
+datetime2#!#datetime2#!#varchar#!#varchar
+1900-01-01 00:59:59.9999120#!#1900-01-01 00:59:59.9999120#!#1900-01-01 00:59:59.999912#!#1900-01-01 00:59:59.999912
+1900-01-01 00:00:01.0000120#!#1900-01-01 00:00:01.0000120#!#1900-01-01 00:00:01.000012#!#1900-01-01 00:00:01.000012
+~~END~~
+
+
+drop table BABEL_2934
+go
+
+CREATE TABLE BABEL_2934 (c DATETIMEOFFSET(7) NULL, d DATETIMEOFFSET(7) NULL)
+GO
+INSERT BABEL_2934 VALUES ('00:59:59.9999123' , NULL)
+GO
+~~ROW COUNT: 1~~
+
+INSERT BABEL_2934 VALUES ('00:00:01.0000120' , NULL)
+go
+~~ROW COUNT: 1~~
+
+
+-- add 300 nanoseconds (the 7th digit shows as '3')
+update BABEL_2934 set d = dateadd(ns, 300, c)
+go
+~~ROW COUNT: 2~~
+
+select c, d, convert(varchar(30), c, 109) as c2, convert(varchar(30), d, 109) as d2 from BABEL_2934
+go
+~~START~~
+datetimeoffset#!#datetimeoffset#!#varchar#!#varchar
+1900-01-01 00:59:59.9999120 +00:00#!#1900-01-01 00:59:59.9999120 +00:00#!#1900-01-01 00:59:59.999912 +00#!#1900-01-01 00:59:59.999912 +00
+1900-01-01 00:00:01.0000120 +00:00#!#1900-01-01 00:00:01.0000120 +00:00#!#1900-01-01 00:00:01.000012 +00#!#1900-01-01 00:00:01.000012 +00
+~~END~~
+
+
+drop table BABEL_2934
+go
+
+select cast('12:15:04.1234567' as TIME(7))
+go
+~~START~~
+time
+12:15:04.123457
```

Comment:
Is this expected?

Also, please add the following cases to MVU tests:
1. `create view v1 as select cast('12:15:04.1234567' as TIME(7))` and `select * from v1`
2. `create function f1 (TIME(7), ...) returns (TIME(7)) ...` and `select f1(...)`
Those are to ensure how customer-created views/functions are impacted by this change during MVU.

Reply:
This is not ideal, but for now it is expected as there is no simple way to override PG's method of converting time strings into datetime values. I will work on the two extra MVU tests